### DISCUSSION TITLE
Deploying Cookie AutoDelete via GPO. How do I duplicate the settings on all browsers?

### DISCUSSION THREAD

**Discussion by max959** [Cookie-AutoDelete/Cookie-AutoDelete#1417] [Q&A / Support]
I have successfully Deployed Cookie AutoDelete via GPO to a host of computers. How do I duplicate the settings on all browsers? Do I copy registry setting from the target machine to the masses?

**Comment by kennethtran93** [comment]
This is currently not possible.  It will need to be manually done on each computer (remote in to manually import).  Relevant feature request is #534 for managed, and #30 for sync.
